app/src/screens/PersonCredential.tsx

```diff
@@ -0,0 +1,166 @@
+import { useAgent } from "@aries-framework/react-hooks"
+import { useNavigation } from "@react-navigation/core"
+import { Button, ButtonType, Screens, useStore, useTheme } from "aries-bifold"
+import React, { ReducerAction, useState, useCallback } from "react"
+import { useTranslation } from "react-i18next"
+import { StyleSheet, Text, View, Image, Dimensions, ImageBackground, TouchableOpacity, Linking } from 'react-native'
+import { SafeAreaView } from 'react-native-safe-area-context'
+import LoadingIcon from "../components/LoadingIcon"
+import { startFlow } from '../helpers/BCIDHelper'
+import { BCDispatchAction, BCState } from "../store"
+
+const PersonCredentialScreen: React.FC = () => {
+
+    const [workflowInFlight, setWorkflowInFlight] = useState<boolean>(false)
+    const { agent } = useAgent()
+    const navigation = useNavigation()
+    const [store, dispatch] = useStore<BCState>()
+
+    const paddingHorizontal = 10
+    const transparent = 'rgba(0,0,0,0)'
+    const borderRadius = 15
+    const borderPadding = 8
+    const { width } = Dimensions.get('window')
+    const cardHeight = width / 2 // a card height is half of the screen width
+    const cardHeaderHeight = cardHeight / 4 // a card has a total of 4 rows, and the header occupy 1 row
+
+    const { ColorPallet, TextTheme } = useTheme()
+    const { t } = useTranslation()
+
+    const styles = StyleSheet.create({
+        pageContent: {
+            marginHorizontal: 20,
+            flexGrow: 1,
+        },
+        pageTextContainer: {
+            display: 'flex',
+            flexDirection: 'row',
+        },
+        container: {
+            backgroundColor: ColorPallet.brand.primaryBackground,
+            height: cardHeight,
+            paddingTop: 15,
+            marginBottom: 20,
+        },
+        outerHeaderContainer: {
+            flexDirection: 'column',
+            backgroundColor: transparent,
+            height: cardHeaderHeight + borderPadding,
+            borderTopLeftRadius: borderRadius,
+            borderTopRightRadius: borderRadius,
+        },
+        innerHeaderContainer: {
+            flexDirection: 'row',
+            marginLeft: borderPadding,
```

Comment:
Does `margin: 8` do the same thing as adding all 4?

Reply:
resolved